ui/src/shared/components/Dropdown.tsx

```diff
@@ -28,35 +54,46 @@ export class Dropdown extends Component {
     disabled: false,
     tabIndex: 0,
   }
+  public dropdownRef: any
 
-  handleClickOutside = () => {
+  constructor(props: Props) {
+    super(props)
+    this.state = {
+      isOpen: false,
+      searchTerm: '',
+      filteredItems: this.props.items,
+      highlightedItemIndex: null,
+    }
+  }
+
+  public handleClickOutside = () => {
```

Comment:
could this be private?

Reply:
the clickoutside component uses it so I think it needs to be public, no?